mod/photos.php

```diff
@@ -1307,9 +1307,9 @@ function photos_content(App $a)
 		}
 
 		$tags = null;
-		$tag_text = Tag::getCSVByURIId($link_item['uri-id']);
 
-		if (!empty($link_item['id']) && !empty($tag_text)) {
+		if (!empty($link_item['id'])) {
+			$tag_text = Tag::getCSVByURIId($link_item['uri-id']);
```

Comment:
Does `$link_item['id']` existence automatically imply `$link_item['uri-id']` existence? I'd prefer if you checked both keys since no other `$link_item` field is used in this condition.

Reply:
When the `id` is present, the `uri-id` will. And if not, then it is a clear bug.